controller/controller_server/decorators.py

```diff
@@ -0,0 +1,21 @@
+import grpc
+from decorator import decorator
+
+from controller.common.utils import set_current_thread_name
+from controller.controller_server.errors import ObjectAlreadyProcessingError
+from controller.controller_server.exception_handler import handle_exception, handle_common_exceptions
+
+
+def csi_method(error_response_type, lock_request_attribute=''):
+    @decorator
+    def handle_csi_method(server_method, servicer, request, context):
+        lock_id = getattr(request, lock_request_attribute, None)
+        set_current_thread_name(lock_id)

```

Comment:
could we also move the `logger.info(controller_method_name)` and ``logger.info("finished {}".format(controller_method_name))`` calls to this decorator?

Reply:
done